datasette/app.py

```diff
@@ -126,6 +126,11 @@
         False,
         "Force URLs in API output to always use https:// protocol",
     ),
+    ConfigOption(
+        "base_url",
+        "",
```

Comment:
Maybe the default should be `/` to try to ensure that the behaviour with this change is in accordance with the way things were. Then the leading slashes that I dropped get put back in by the `<base>` tag in case you don't specify one in the config. The downside(?) is that your pages always have a base tag.  I used to use `<base>` a lot back in 1995 or so, to host multiple domains/sites on one server, before the advent of the `Host` header, etc.